packages/SwingSet/src/kernel/kernel.js

```diff
@@ -703,14 +722,21 @@ export default function buildKernel(kernelEndowments) {
             metered: false,
             vatPowerType: 'static',
             allowSetup: true, // TODO: only needed by comms, disallow elsewhere
+            vatOptions,
+            creationOptions,
           }));
-      addVatManager(vatID, manager, options);
+      addVatManager(vatID, manager, vatOptions, creationOptions);
+    }
+
+    function createVatDynamicallyByName(bundleName, options) {
+      return createVatDynamically(getBundle(bundleName), options);
```

Comment:
let's add a check here to throw if the bundleName is not in the table, I think that's an easy error to trigger from userspace so we should give a clear diagnostic, rather than having `createVatDynamically` emit something obscure when `importBundle` tries to deal with `undefined`

Reply:
Good catch, er, so to speak.  Will do.